charts/pulsar/values.yaml

```diff
@@ -856,16 +856,16 @@ grafana:
     targetPort: 3000
     annotations: {}
   plugins: []
+  ## Grafana configMap
+  ## templates/grafana-configmap.yaml
+  ##
+  configData: {}
   ## Grafana ingress
   ## templates/grafana-ingress.yaml
   ##
   ingress:
     enabled: false
-    annotations:
```

Comment:
why did you remove the default values?

Reply:
They get merged with other values if a user provides annotations. This also matches the other ingress values. So if I am not using nginx ingress, it still attaches the nginx ingress annotations.